spec/orders_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'orders'
+# require 'sms'
+
+describe Orders do
+
+  let(:menu) { double :menu, :menu_items => { :pie => 4.5, :chips => 2.50 } }
+  let(:mockbill) { double :mockbill, :bill => 14 }
+  let(:bill_klass) { double :bill_klass, :new => mockbill }
+
+  it 'chooses the menu' do
+    expect(subject.choose_menu(menu)).to eq menu.menu_items
+  end
+
+  it 'responds to 2 arguments' do
+    expect(subject).to respond_to(:cust_order).with(2).arguments
+  end
+
+  it "can place multiple orders" do
+    subject.choose_menu
+    subject.cust_order("pie", 3)
+    subject.cust_order("chips", 2)
+    expect(subject.cust_orders).to eq({ pie: 3, chips: 2 })
+  end
+
+  it "raises an error if dish is not on the menu" do
+    subject.choose_menu
+    expect { subject.cust_order('chicken', 2) }.to raise_error "sorry, dish is not available"
+  end
+
+  it "confirms customer order" do
+    subject.choose_menu
+    subject.cust_order("pie", 2)
+    subject.cust_order("chips", 2)
+    expect(subject.order_conf).to eq({ :pie => 2, :chips => 2 })
+  end
+
+  it "returns the bill" do
+    subject.choose_menu
+    subject.cust_order("pie", 2)
+    subject.cust_order("chips", 2)
+    expect(subject.bill(bill_klass)).to eq 14
```

Comment:
This test is probably a little more in depth that it actually needs to be.

Returning the total price is the responsibility of the `bill` method in the `Bill` class, the `bill` method in the `Order` class is just responsible for calling that - it doesn't know or care (and it shouldn't know or care) about what it actually does.

A better test here would be something like 
```ruby
it 'returns the bill' do
  expect(mockbill).to receive(:bill)
  subject.boll(bill_klass)
end
```